docs/topics/mta-cli-run.adoc

```diff
@@ -0,0 +1,133 @@
+
+[source,terminal,subs="attributes+"]
+----
+
+$ <{ProductShortName}_HOME>/bin/{mta-cli} analyze --mode source-only --input /path/to/seam-booking-5.2/ \
+    --output /path/to/report-output/ --target eap6 --packages org.jboss.seam
+----
+[]
+[discrete]
+=== Running cloud-readiness rules
+
+The following command analyzes the `com.acme` and `org.apache` packages of the link:https://github.com/windup/windup/blob/master/test-files/jee-example-app-1.0.0.ear[jee-example-app-1.0.0.ear] example EAR archive for migrating to JBoss EAP 7. It also evaluates for cloud readiness:
+
+[source,terminal,subs="attributes+"]
+----
+$ <{ProductShortName}_HOME>/{mta-cli} analyze --input /path/to/jee-example-app-1.0.0.ear \
+    --output /path/to/report-output/ \
+    --target eap7
+----
+[]
+////
+[discrete]
+=== Overriding {ProductShortName} properties
+
+To override the default _Fernflower_ decompiler, pass the `-Dwindup.decompiler` argument on the command line. For example, to use the _Procyon_ decompiler, use the following syntax:
+
+[source,terminal,subs="attributes+"]
+----
+$ <{ProductShortName}_HOME>/{mta-cli} -Dwindup.decompiler=procyon \
+    --input <INPUT_ARCHIVE_OR_DIRECTORY> --output <OUTPUT_REPORT_DIRECTORY> \
+----
+////
+////
+[id="cli-bash-completion_{context}"]
+== {ProductShortName} {CLINameTitle} Bash completion
+
+The {ProductShortName} {CLIName} provides an option to enable Bash completion for Linux systems, allowing the {ProductShortName} command-line arguments to be auto completed by pressing the Tab key when entering the commands. For instance, when Bash completion is enabled, entering the following displays a list of available arguments.
+
+[source,terminal,subs="attributes+"]
+----
+$ <{ProductShortName}_HOME>/{mta-cli} [TAB]
+----
+
+[discrete]
+[id="bash-completion-temporary_{context}"]
+=== Enabling Bash completion
+
+To enable Bash completion for the current shell, execute the following command:
+
+[source,terminal,subs="attributes+"]
+----
+$ source <{ProductShortName}_HOME>/bash-completion/{mta-cli}
+----
+
+[discrete]
+[id="bash-completion-persistent_{context}"]
+=== Enabling persistent Bash completion
+
+The following commands allow Bash completion to persist across restarts:
+
+* To enable Bash completion for a specific user across system restarts, include the following line in that user's `~/.bashrc` file.
++
+[source,terminal,subs="attributes+"]
+----
+source <{ProductShortName}_HOME>/bash-completion/{mta-cli}
+----
+
+* To enable Bash completion for all users across system restarts, copy the {ProductName} {CLIName} Bash completion file to the `/etc/bash_completion.d/` directory as the root user.
++
+[source,terminal,subs="attributes+"]
+----
+# cp <{ProductShortName}_HOME>/bash-completion/{mta-cli} /etc/bash_completion.d/
+----
```

Comment:
The bash-completion folder doesn't exist in new mta-cli version, this section should be removed

Reply:
Was commented out, but let us delete